main/tests/test_orders.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+from unittest import TestCase
+from views.orders import order_id,orderlist, OnlineRestuarant
+
+class OnlineRestuarantTest(unittest.TestCase):
+
+    def test_welcome(self):
+        self.assertTrue(True)
+
+    def test_get_orders(self):
+        if orderlist == []:
```

Comment:
I think this if statement should  be part of the code you are testing not part of the test...
So in otherwords just write a test to check whether  value error is actually raised..

Reply:
@kaggwachristopher Thanx for the advise. let me apply it and see the results